design-dimensions/README.md

```diff
@@ -0,0 +1,66 @@
+|------------------------------------|-----------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Server mediated                    | Whether the API makes use of a server to compute the output                                         | PCM: NoE-ARA: No<br>A-ARA: Yes<br>IPA: Yes<br>SKAN: Yes                                                                                                                                                                                                                                                           |
+| Privacy definition of API output   | [Differential privacy](https://en.wikipedia.org/wiki/Differential_privacy), information theoretic (e.g. [entropy](https://en.wikipedia.org/wiki/Entropy_(information_theory))), [k-anonymity](https://en.wikipedia.org/wiki/K-anonymity)                             | PCM: information theoretic<br>E-ARA: local DP + information theoretic<br>A-ARA: central DP + information theoretic<br>IPA: central DP<br>SKAN: Combination of information theoretic + k-anonymity|
+| On-device / off-device attribution | Whether join/attribution occurs on-device or in a server                                            | PCM: device<br>E-ARA: device<br>A-ARA: device<br>IPA: server<br>SKAN: device                                                                                                                                                                                                                                      |
+| Where is budgeting applied         | Where are user contributions bounded within the above scope?                                        | PCM: on-device<br>E-ARA: on-device<br>A-ARA: hybrid on-device and in servers<br>IPA: servers<br>SKAN: on-device                                                                                                                                                                                                   |
+| Scope of privacy budgeting         | If applicable, the axes along which privacy budgeting applies: time epoch, site, campaign, delegate | PCM: Per device/source-destination site pair. Limited in rate by reporting delay and user interaction.<br>A-ARA: Per epoch, source ⇔ destination site pair, device.<br>E-ARA: Per epoch, source ⇔ destination site pair, device<br>IPA: Per week/epoch, per site, per match key.<br>SKAN: Per device / source app |
```

Comment:
```suggestion
| Scope of privacy budgeting         | If applicable, the axes along which privacy budgeting applies: time epoch, site, campaign, delegate | PCM: Per device/source-destination site pair. Limited in rate by reporting delay and user interaction.<br>A-ARA: Per epoch, source ⇔ destination site pair, device.<br>E-ARA: Per epoch, source ⇔ destination site pair, device<br>IPA: Per week/epoch, per site<br>SKAN: Per device / source app |
```

@eriktaubeneck check my work please, but I believe that we've dispensed with a per-match key budget.  We're now looking at query limiting (and maybe public query logs).

Reply:
"match key" here is standing in for the user. How are user contributions bounded in IPA if not by match key?

Reply:
You are both right. Contributions are bound by match key (as a stand-in for user), but the budget it tracked globally, based on the worst case assumption that every match key is included in every query (which is what @martinthomson is referring to.)

Looking back at this, I'm not sure which the intent is here, but I am assuming the former. It might just be worth calling out that the scope is per match key, but it's tracked by number of queries?

Reply:
The intention with this row was to document abstractly the scope of privacy, in the differential privacy "privacy unit" sense, and less about the technical details about how we do it (usually a combination of contribution bounding and rate limiting).

Reply:
In that case, I think this is accurate and doesn't need the suggestion. @martinthomson feel free to open a new issue if you think it's worth including the "how".